packages/app/src/Element/Note.tsx

```diff
@@ -175,6 +176,16 @@ export default function Note(props: NoteProps) {
     }
   }, [inView, entry, extendable]);
 
+  function goToReplaceableEvent(e: React.MouseEvent, ev: NEvent) {
+    e.stopPropagation();
+    const d = ev.Tags.find(t => t.Key === "d")?.Original[1];
+    if (d) {
+      const href = encodeTLV(d, NostrPrefix.Address, [], ev.PubKey, ev.Kind);
+      window.open(`nostr:${href}`);
+      //window.open(`https://habla.news/a/${href}`);
```

Comment:
not sure if any client support naddr atm, perhaps i should not make it clickable @v0l ?

Reply:
I think it's fine to use the habla.news link. You could allow user to specify choose a different client in user preferences if people complain about it.